addon/-private/system/model/internal-model.js

```diff
@@ -284,7 +286,15 @@ export default class InternalModel {
         createOptions.container = store.container;
       }
 
+      let recordData = this._recordData;
```

Comment:
I don't see anywhere that `recordData` has the potential to be reassigned. Does ember-data have a policy on `let` vs `const`? 

There are advantages in TS to using const, in that guards can effectively narrow types, since no branch of the code could alter the type of the variable through reassignment.

Reply:
@igort thoughts? Overall we’ve preferred let to const until now, and for JS code we’ve definitely felt there was more signal when doing so.

Reply:
Yeah, we use `const` only for constants.